src/components/common/Routing.tsx

```diff
@@ -1,11 +1,14 @@
 import { BrowserRouter, Route } from 'react-router-dom';
 
 import Home from '@pages/Home';
+import MyPage from '@pages/MyPage';
 
 function Routing() {
 	return (
 		<BrowserRouter>
 			<Route path="/home" component={Home} exact={true} />
+			<Route path="/" component={Home} exact={true} />
+			<Route path="/my-page" component={MyPage} exact={true} />
```

Comment:
사소하지만 exact로만 사용해도 되지 않을까요?
```suggestion
			<Route path="/" component={Home} exact />
			<Route path="/my-page" component={MyPage} exact />
```

Reply:
저도 버튼의 경우 py 적용이 되지 않아서 Height를 사용했었습니다..! 혹시 알게 되시면 공유 부탁드립니다

Reply:
어제 엠제이네 팀에서 동일한 문제가 있었던 걸로 알아서 이따 물어볼게요~!